web/beacon-app/src/application/routes/root.tsx

```diff
@@ -42,13 +43,15 @@ const router = createBrowserRouter(
         <Route path="organization" element={<OrganizationPage />} />
         <Route path="profile" element={<MemberDetailsPage />} />
       </Route>
-      <Route element={<MainLayout />}>
+      <Route element={<PublicRoutes />}>
```

Comment:
would you mind renaming `PublicRoutes` to AuthedRoutes or any naming that is reflecting the action?

Reply:
PublicRoutes is a wrapper for unauthenticated routes. It's somewhat the opposite of `PrivateRoutes` which contains all authenticated routes. Change it by AuthedRoutes can be confusing.

Reply:
Since those routes can be available whether the user is authenticated or not, I consider them as `common` not `public`.